File: University Student Rating/Code/androidclient/UniversityStudentRating/app/src/main/java/com/example/arturrinkis/universitystudentrating/DTO/Rating.java
package com.example.arturrinkis.universitystudentrating.DTO;

import java.util.Date;

public class Rating {
    private int id;

    private String name;

    private int points;

    private Date date;

    private int courseId;

    private int disciplineId;

    private Integer studentId;

    private Integer professorId;

    private Integer ratingDivisionId;

    private Course course;

    private Discipline discipline;

    private RatingDivision ratingDivision;

    private UserProfile userProfile;


    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getPoints() {
        return points;
    }

    public void setPoints(int points) {
        this.points = points;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public int getCourseId() {
        return courseId;
    }

    public void setCourseId(int courseId) {
        this.courseId = courseId;
    }

    public int getDisciplineId() {
        return disciplineId;
    }

    public void setDisciplineId(int disciplineId) {
        this.disciplineId = disciplineId;
    }

    public Integer getStudentId() {
        return studentId;
    }

    public void setStudentId(Integer studentId) {
        this.studentId = studentId;
    }

    public Integer getProfessorId() {
        return professorId;
    }

    public void setProfessorId(Integer professorId) {
        this.professorId = professorId;
    }

    public Integer getRatingDivisionId() {
        return ratingDivisionId;
    }

    public void setRatingDivisionId(Integer ratingDivisionId) {
        this.ratingDivisionId = ratingDivisionId;
    }

    public Course getCourse() {
        return course;
    }

    public void setCourse(Course course) {
        this.course = course;
    }

    public Discipline getDiscipline() {
        return discipline;
    }

    public void setDiscipline(Discipline discipline) {
        this.discipline = discipline;
    }

    public RatingDivision getRatingDivision() {
        return ratingDivision;
    }

    public void setRatingDivision(RatingDivision ratingDivision) {
        this.ratingDivision = ratingDivision;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public UserProfile getUserProfile() {
        return userProfile;
    }

    public void setUserProfile(UserProfile userProfile) {
        this.userProfile = userProfile;
    }
}

File: University Student Rating/Code/server/UniversityStudentRatingService/DataAccessLayer/DbContext/Rating.cs
namespace DataAccessLayer.DbContext
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("Rating")]
    public partial class Rating
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public int Points { get; set; }

        [Column(TypeName = "date")]
        public DateTime Date { get; set; }

        public int CourseId { get; set; }

        public int DisciplineId { get; set; }

        public int? StudentId { get; set; }

        public int? ProfessorId { get; set; }

        public int? RatingDivisionId { get; set; }

        public virtual Course Course { get; set; }

        public virtual Discipline Discipline { get; set; }

        public virtual RatingDivision RatingDivision { get; set; }

        public  UserProfile UserProfile { get; set; }

        public  UserProfile UserProfile1 { get; set; }
    }
}
